roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -121,6 +121,13 @@ server {
     deny all;
   }
   {% endblock %}
+  
+  {% block blade_twig_templates -%}
+  # Prevent Blade and Twig templates from being accessed directly.
+  location ~* \.(blade.php|twig)$ {
```

Comment:
Shouldn't the `.` be escaped? `blade\.php`? Or does it not matter?

Reply:
You're right, it should've been. Accidentally overlooked it and didn't catch it during testing due to the period matching any character.